src/main/groovy/org/shipkit/internal/gradle/InitConfigFileTask.java

```diff
@@ -13,6 +17,13 @@
     private static final Logger LOG = Logging.getLogger(InitConfigFileTask.class);
 
     private File configFile;
+    private GitOriginRepoProvider gitOriginRepoProvider;
+    public static final String FALLBACK_GITHUB_REPO = "mockito/mockito-release-tools-example";
+
+    public InitConfigFileTask(){
+        ProcessRunner runner = new DefaultProcessRunner(getProject().getProjectDir());
```

Comment:
I actually do, I can pass only projectDir to GitOriginRepoProvider, but it's easier to test GitOriginRepoProvider if I pass ProcessRunner directly.